.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
 - nvm install 8.0.0
 install:
 - travis_retry ./ops/travis/travis-install.sh $JOB
-- export PYTHONPATH=${PYTHONPATH}:${HOME}/google_appengine
+- export PYTHONPATH=${PYTHONPATH}:${HOME}/google_appengine:${HOME}/google_appengine/lib/protorpc-1.0
```

Comment:
If we can figure out how to get everything from that `lib` folder on the path, that would be hella